package.json

```diff
@@ -70,6 +70,7 @@
     "istanbul": "*",
     "microtime": "~0.6.0",
     "mocha": "2.3.4",
+    "npm": "2",
```

Comment:
I can pull it into a different branch... without this clingwrap tries to install npm 3, which we don't want